tests/semantic_analyser.cpp

```diff
@@ -3328,9 +3364,9 @@ TEST(semantic_analyser, string_size)
        0);
   stmt = driver.ctx.root->probes.at(0)->stmts.at(0);
   map_assign = dynamic_cast<ast::AssignMapStatement *>(stmt);
-  ASSERT_EQ(map_assign->map->key_type.size(), 14UL);
-  ASSERT_TRUE(map_assign->map->key_type.args_.at(0).IsStringTy());
-  ASSERT_EQ(map_assign->map->key_type.args_.at(0).GetSize(), 6UL);
+  ASSERT_EQ(map_assign->map->key_type.GetSize(), 16UL);
+  ASSERT_TRUE(map_assign->map->key_type.IsTupleTy());
+  ASSERT_EQ(map_assign->map->key_type.GetSize(), 16UL);
```

Comment:
```suggestion
  ASSERT_EQ(map_assign->map->key_type.GetSize(), 16UL);
```
This line is checking the same as the line two above it now instead of checking the string size